### PR TITLE
 Add result URL validator, and more validators(#179)

### LINKED ISSUE #179: Add result URL validator
add validation to Form.result_url and autocomplete
url must be only google doc
docs.google.com
Andrii, u can use url parser: from urllib.parse import urlparse

### PR DESCRIPTION
**Pull request by AndriiKret**
- Added Setting_autocomplete validation
- Added Checkbox range to choice options amount POST validation
- Added Form Result URL validator

### CODE REVIEW

File: src/app/schemas/field.py [hunk 1]
```diff
@@ -69,6 +69,31 @@ class Meta:
...
+            if min_value is not None:
+                if min_value < 0:
+                    raise ValidationError('Minimal selective options must be positive')
+                if min_value > len(options_list):
+                    raise ValidationError('Min selective options must be less than list of options')
+            if max_value is not None:
+                if max_value < 0:
+                    raise ValidationError('Maximal selective options must be positive')
+                if max_value > len(options_list):
+                    raise ValidationError('Max selective options must be less than list of options')
```
Comment (dtymkiv) [inline src/app/schemas/field.py]: Use "Min" or "Minimal", not both

File: src/app/schemas/setting_autocomplete.py [hunk 2]
```diff
@@ -21,3 +24,34 @@ class Meta:
...
+    def validate_data_url(self, data, **kwargs):
+        """
+        Validates url, which must be docs.google.com
+        :param data:
+        :param kwargs:
+        :return:
+        """
+        parsed_url = urlparse(data.get('data_url'))
+        if parsed_url.netloc != 'docs.google.com':
+            raise ValidationError('Wrong URL entered')
```
Comment (dtymkiv) [inline src/app/schemas/setting_autocomplete.py]: Exactly the same method as in schemas/forms.py

File: src/app/schemas/field.py (hunk 1 above)
Comment (AndriiKret) [reply src/app/schemas/field.py]: fixed

File: src/app/schemas/setting_autocomplete.py (hunk 2 above)
Comment (AndriiKret) [reply src/app/schemas/setting_autocomplete.py]: fixed

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/schemas/field.py [hunk 3]
```diff
@@ -69,6 +69,31 @@ class Meta:
...
+            min_value = range_dict.get('min')
+            max_value = range_dict.get('max')
+            if min_value is not None:
+                if min_value < 0:
+                    raise ValidationError('Min selective options must be positive')
+                if min_value > len(options_list):
+                    raise ValidationError('Min selective options must be less than list of options')
+            if max_value is not None:
+                if max_value < 0:
+                    raise ValidationError('Min selective options must be positive')
```
Comment (dtymkiv) [inline src/app/schemas/field.py]: Max)

**Review by dtymkiv — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Review by yurdosii — APPROVED** [review]

**Pull request merged**